runtime/src/lib.rs

```diff
@@ -146,16 +146,21 @@ macro_rules! create_zeitgeist_runtime {
             {
                 Balances: pallet_balances::{Call, Config<T>, Event<T>, Pallet, Storage},
                 Currency: orml_currencies::{Call, Event<T>, Pallet, Storage},
-                Orderbook: zrml_orderbook_v1::{Call, Event<T>, Pallet, Storage},
-                PredictionMarkets: zrml_prediction_markets::{Call, Event<T>, Pallet, Storage},
                 RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Call, Pallet, Storage},
                 Sudo: pallet_sudo::{Call, Config<T>, Event<T>, Pallet, Storage},
-                Swaps: zrml_swaps::{Call, Event<T>, Pallet, Storage},
                 System: frame_system::{Call, Config, Event<T>, Pallet, Storage},
                 Timestamp: pallet_timestamp::{Call, Pallet, Storage, Inherent},
                 Tokens: orml_tokens::{Config<T>, Event<T>, Pallet, Storage},
                 TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
 
+                // Zeitgeist
+
+                Court: zrml_court::{Event<T>, Pallet, Storage},
+                MarketCommons: zrml_market_commons::{Pallet, Storage},
+                Orderbook: zrml_orderbook_v1::{Call, Event<T>, Pallet, Storage},
+                PredictionMarkets: zrml_prediction_markets::{Call, Event<T>, Pallet, Storage},
```

Comment:
Requires increment of `spec_version` and `transaction_version` before merge

Reply:
Why remove the explicit indices on the pallets?

Reply:
Bad re-base. Indices were re-introduced